docs/guides/dns-over-https.md

```diff
@@ -35,7 +35,18 @@ cloudflared -v
 
 **Note:** Binaries for other operating systems can be found here: <https://developers.cloudflare.com/argo-tunnel/downloads/>
 
-#### ARM architecture (Raspberry Pi)
+#### arm64 architecture (64-bit Raspberry Pi)
+
+Get the download link of `cloudflared-linux-arm64` from <https://github.com/cloudflare/cloudflared/releases/>
```

Comment:
This seems superfluous now that you have the correct automatic link to source the most recent binary below

Reply:
thanks. deleted